toonz/sources/toonzlib/preferences.cpp

```diff
@@ -255,6 +255,8 @@ Preferences::Preferences()
     , m_fitToFlipbookEnabled(false)
     , m_previewAlwaysOpenNewFlipEnabled(false)
     , m_autosaveEnabled(false)
+    , m_autosaveSceneEnabled(true)
+    , m_autosaveOtherFilesEnabled(true)
```

Comment:
IMO the initial value of `m_autosaveOtherFilesEnabled` should be `false` since this feature originally did not save non-scene files. Setting the initial value of this option to "enabled" may cause unexpected behavior for some users.